integration-tests/src/test/scala/za/co/absa/spline/harvester/SparkUnimplementedCommandsSpec.scala

```diff
@@ -44,20 +47,18 @@ class SparkUnimplementedCommandsSpec extends FlatSpec
   val databaseName = "testDb"
   val tableName = "testTable"
 
-  private val tempDirPath = TempDirectory(prefix = "test").deleteOnExit().path.toFile.getAbsolutePath
-
   "Lineage for create database" should "be caught" in
     withCustomSparkSession(_
       .enableHiveSupport()
       .config("hive.exec.dynamic.partition.mode", "nonstrict")) { spark =>
-        try
-          withLineageTracking(spark) { lineageCaptor =>
-            val (plan, _) = lineageCaptor.lineageOf {
-              spark.sql(s"CREATE DATABASE $databaseName") // CreateDatabaseCommand
-            }
+
+      withRestartingSparkContext {
```

Comment:
`withRestartingSparkContext` method is designed to be used _before_ `withCustomSparkSession` because it kills the session and effectively undoes all the changes to the session in all previous methods.
So it this test Hive is not actually enabled. Do you actually need it considering that the test passes without Hive?

Reply:
I used database fixture instead, that should be quicker and take care of the cleaning.
The Hive is actually not needed in this case, so I removed it, but in most of other tests it's necessary and also useful for creating the tests.